diff --git a/jsx-runtime/index.d.ts b/jsx-runtime/index.d.ts
index af5ffe2ac243671f5aa555f8f7ca265aa74e5af0..5283994831376c8483c727c4c560f72c3a3b3b35 100644
--- a/jsx-runtime/index.d.ts
+++ b/jsx-runtime/index.d.ts
@@ -9,7 +9,7 @@ import type { NativeElements, ReservedProps, VNode } from '@vue/runtime-dom'
 export { h as jsx, h as jsxDEV, Fragment, h as jsxs } from '@vue/runtime-dom'
 
 export namespace JSX {
-  export interface Element extends VNode {}
+  // export interface Element extends VNode {}
   export interface ElementClass {
     $props: {}
   }
